docs/containers-storage.conf.5.md

```diff
@@ -59,6 +59,11 @@ A common use case for this field is to provide a local storage directory when us
   container storage run dir (default: "/run/containers/storage")
 Default directory to store all temporary writable content created by container storage programs. The rootless runroot path supports environment variable substitutions (ie. `$HOME/containers/storage`)
 
+**driver_priority**=[]
+  Priority list for the storage drivers that will be tried in case no driver is set.
```

Comment:
"will be tried" is a bit vague.  Please be specific when a certain driver will be used and when not.

Reply:
Does it make more sense now?